src/app/templates/activity/activity-item/hooks/use-activity-item-view-model.tsx

```diff
@@ -84,9 +121,11 @@ export const useActivityItemViewModel = (activity: DisplayableActivity) => {
     }
 
     if (isInteractionActivityTypeGuard(activity)) {
+      const width = isDefined(activity.subtype) && isQuipuswapSubtype(activity.subtype) ? 36 : 24;
+
       return isDefined(activity.subtype) && !isAllowanceInteractionActivityTypeGuard(activity) ? (
-        <div className={classNames('flex items-center justify-center', actorAvatarStyles)}>
-          <img src={interactionActivitiesLogos[activity.subtype]} alt={activity.subtype} height={12} width={24} />
+        <div className={classNames('flex items-center justify-center overflow-hidden', actorAvatarStyles)}>
+          <img src={interactionActivitiesLogos[activity.subtype]} alt={activity.subtype} height={12} width={width} />
```

Comment:
Please check the look of this element. If everything is fine, just resolve this conversation

Reply:
image is square, but need to be with rounded edges.. this property is for this. Check `actorAvatarStyles`